src/covid19sim/configs/test_config.yml

```diff
@@ -9,7 +9,7 @@ SHOULD_MODIFY_BEHAVIOR : True
 USE_ORACLE: False
 
 ## INTERVENTIONS
-INTERVENTION_DAY : 10
+INTERVENTION_DAY : 2
```

Comment:
should this change be here?

Reply:
I wanted to speed stuff up, so I reduced some of the values while maintaining the core functionality of the tests.